_posts/en/newsletters/2020-04-01-newsletter.md

```diff
@@ -0,0 +1,132 @@
+  then further delegate the spending authority to a third user (Carol),
+  etc.  The offchain delegation operations are all performed with the
+  cooperation of a trusted third party who can only steal funds if they
+  collude with a delegated signer (such as previous delegates Alice or
+  Bob).  A delegated signer can always spend the UTXO onchain without
+  needing permission from the trusted third party, arguably making
+  statechains less trusted than federated [sidechains][topic sidechains] (all other
+  things being equal).  Because anyone who was ever a delegate can
+  trigger an onchain spend, statechains were designed to use the
+  [eltoo][topic eltoo] mechanism to ensure an onchain spend by the most
+  recent delegate (Carol) can take precedence over spends by previous
+  delegates (Alice and Bob), assuming the trusted third party hasn't
+  colluded with a previous delegate to cheat.
+
+    This week, Tom Trevethan [posted][trevethan statechains] to the
+    Bitcoin-Dev mailing list about about two modifications of the
+    statechain design that could allow it to be used with the current
+    Bitcoin protocol rather than waiting for proposed soft fork changes
+    such as [schnorr signatures][topic schnorr signatures] signatures
+    and [SIGHASH_ANYPREVOUT][topic sighash_noinput]:
+
+    1. Replace the eltoo mechanism (which requires either [BIP116][]
+       `SIGHASH_NOINPUT` or [bip-anyprevout][] `SIGHASH_ANYPREVOUT`)
+       with a decrementing locktime similar to that proposed for
+       [duplex micropayment channels][].  E.g., when Alice receives
+       control over a statechain UTXO, a timelock would prevent her from
+       being able to unilaterally spend it onchain for 30 days; when
+       Alice transfers the UTXO to Bob, a timelock would restrict him
+       for only 29 days---this gives a spend by Bob precedence over a
+       spend by Alice.  The downside of this approach is that delegates
+       might need to wait a long time before being able to spend their
+       funds without permission from the trusted third party.
+
+    2. Replace the 2-of-2 schnorr multisig between the trusted third
+       party and the current delegate (using an [adaptor
+       signature][scriptless scripts]) with a single-sig using [secure
+       multiparty computation][mpc].  The main downside of this approach
+       is an increased complexity that makes security review harder.
+
+    Several people replied to the thread with comments and suggested
+    alternatives.
+
+- **Mitigating differential power analysis in schnorr signatures:**
+  Lloyd Fournier started a [discussion][fournier dpa] on the Bitcoin-Dev
+  mailing list about the proposal described in [Newsletter #87][news87
+  bip340 update] to update the [BIP340][] specification for schnorr
+  signatures with a recommended nonce generation function that claims to
+  be more resistant against [differential power analysis][dpa].  Power
```

Comment:
"Power analysis means monitoring ...." feels a little clumsy. Is "A power analysis attack involves monitoring..." better?